src/EPDWifi.h

```diff
@@ -21,22 +21,31 @@
   */
 #pragma once
 #include <WiFi.h>
+#include <WiFiMulti.h>
 
 /* Start and connect to the wifi */
 bool StartWiFi(int &rssi) 
 {
    IPAddress dns(8, 8, 8, 8); // Google DNS
-   
+
+   WiFiMulti wifiMulti;
+
    WiFi.mode(WIFI_STA);
    WiFi.disconnect();
    WiFi.setAutoConnect(true);
    WiFi.setAutoReconnect(true);
 
-   Serial.print("Connecting to ");
-   Serial.println(WIFI_SSID);
-   delay(100);
-   
-   WiFi.begin(WIFI_SSID, WIFI_PW);
+/* add AP's here with AP Name, AP Password pairs */
```

Comment:
I would not just add plain credentials here but use the same mechanism as before. Aka in the `ConfigTemplate.h` you add for example two or three entries:
```cpp
#define WIFI_SSID1       "your wifi ssid"
#define WIFI_SSID1_PWD       "your wifi pwd"
#define WIFI_SSID2       "your wifi ssid"
#define WIFI_SSID2_PWD       "your wifi pwd"
```
and then we can use the `ifdef` construct for compilation. Like
```cpp
#ifdef WIFI_SSID1 
wifiMulti.addAP(WIFI_SSID1 , WIFI_SSID1_PWD);
#endif
#ifdef WIFI_SSID2
wifiMulti.addAP(WIFI_SSID2 , WIFI_SSID2_PWD);
#endif
```

Reply:
Yeah, I really, really didn't want to do that as it's such an inelegant way to code this.  I really wanted a #define construct that would let me define an array of access points, but I couldn't figure out how to do that.  I'll implement the suggestion, and allow up to 10 access points.